xamarin/Xamarin.Forms
# [Core] Removed null values from the DisplayActionSheet method

### Description of Change ###

Removed null values from the DisplayActionSheet method.

### Issues Resolved ### 
- fixes #4744

### API Changes ###
 None

### Platforms Affected ### 
- Core/XAML (all platforms)
- iOS
- Android
- UWP

### Behavioral/Visual Changes ###
User would not see the button for null value when calling DisplayActionSheet method by passing null values in the button array.

### Before/After Screenshots ### 
Not applicable

### Testing Procedure ###
- Open Issue4744 in UWP app.
- Click on "Display action sheet with null" value button.
- You should see that null value buttons are not showing now.

### PR Checklist ###
